docs/USAGE.md

```diff
@@ -55,6 +56,35 @@ There are a number of strategies available to provide [secure remote access]({{b
 
 If you want to use openHAB Android on a wall mounted tablet, go to settings and select `Disable display timer` and `Fullscreen`.
 
+## Alarm Clock
+
+The openHAB app will send the next wake-up time from your alarm clock app to the server. Please configure an Item name in the settings.
+
+Example item definition:
+```
+Number AndroidAlarm
+
+```
+
+Example rule:
+```
+rule "Alarm Trigger"
+when
+    Time cron "*/10 * * * * ?"
+then
+    if (AndroidAlarm.state as Number == 0) {
+        // Alarm is turned off
+        return
+    }
+    val diff = AndroidAlarm.state as Number - now().millis
+    if (diff <= 15000) {
+        // Turn on stuff
```

Comment:
What does this comment mean? It's a bit unclear what exactly the if condition is supposed to mean, at least if you don't know the implementation)